discord-webhook.ts

```diff
@@ -53,6 +58,14 @@ async function run() {
 	assertEnv('SUITE', env.SUITE)
 	assertEnv('STATUS', env.STATUS)
 	assertEnv('DISCORD_WEBHOOK_URL', env.DISCORD_WEBHOOK_URL)
+	const isRolldownVite = !!env.IS_ROLLDOWN_VITE
```

Comment:
we don't test with rolldown-vite here but took the change anyways so the next sync to vite-ecosystem-ci is smaller and maybe we want to start testing with it in the future